diff --git a/course/view.php b/course/view.php
index 6acd0f8..c2058e8 100644
--- a/course/view.php
+++ b/course/view.php
@@ -7,6 +7,11 @@
     require_once($CFG->libdir.'/conditionlib.php');
     require_once($CFG->libdir.'/completionlib.php');
 
+    // TURFORLAG CORE HACK
+    $PAGE->requires->jquery();
+    $PAGE->requires->jquery_plugin('ui');
+    // END TURFORLAG CORE HACK
+
     $id          = optional_param('id', 0, PARAM_INT);
     $name        = optional_param('name', '', PARAM_RAW);
     $edit        = optional_param('edit', -1, PARAM_BOOL);
